scalastyle-config.xml

```diff
@@ -88,17 +88,6 @@
     ]]></customMessage>
   </check>
 
-  <check customId="classforname" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
```

Comment:
OOC, why are deleting this instead of setting `// scalastyle:off classforname`?

Reply:
`Utils.classForName` is private to Spark.